spec/features/requests/request_spec.rb

```diff
@@ -639,6 +640,22 @@
           expect(confirm_email.html_part.body.to_s).to have_content("La mirada : looking at photography in Latin America today")
         end
 
+        context 'disavowed user' do
+          it 'allows a non circulating item with not item data to be digitized' do
+            stub_request(:get, patron_url)
```

Comment:
I think the title needs to be different that it does not allow a digitization request.